screens/AboutScreen.js

```diff
@@ -1,28 +1,55 @@
 import React from "react";
-import { ScrollView, View, Text, StyleSheet, Image } from "react-native";
+import {
+  Image,
+  Linking,
+  ScrollView,
+  StyleSheet,
+  Text,
+  View,
+} from "react-native";
 import Constants from "expo-constants";
 import colours from "../utils/colours";
 
 const AboutScreen = () => {
+  // Dark background, light text
   const darkSectionStyle = addStyleTo("sectionContainer", {
     backgroundColor: colours.brand,
   });
 
-  const lightSectionStyle = addStyleTo("sectionContainer", {
-    backgroundColor: colours.backgroundGrey,
-  });
-
-  const lightSectionTextStyle = addStyleTo("sectionText", {
-    color: "white",
+  const darkSectionTitleStyle = addStyleTo("sectionTitle", {
```

Comment:
As an FYI, feel free to update this pattern in future, it was just a quick thing I added to get the about page out quick. We're probably starting to overload it a bit now. Maybe we should spike out whether styled components works well with react-native?

Reply:
No worries! I hadn't seen this pattern before so that's cool. 🤩 

Agree about code quality – I've been sacrificing it a little bit for the sake of moving fast, but I can spend some time later this week refactoring. @mattjcheah noticed we have both `package-lock.json` and `yarn.lock` files (my bad) so that's another lil small fix to make.